lib/school_house_web/live/conferences_live.html.heex

```diff
@@ -14,8 +14,8 @@
 <div class="bg-body dark:bg-body-dark">
   <div class="md:w-11/12 lg:w-8/12 mx-auto">
     <.form let={f} for={:filters} id="filter-form" class="flex-row flex items-center mb-6 px-2 text-primary dark:text-primary-dark" phx-change="filter">
-      <p class="text-light dark:text-light-dark mr-4"><%= gettext("Filter")%></p>
-      <div class="mx-2">
+      <p class="text-light dark:text-light-dark mr-5"><%= gettext("Filter")%></p>
+      <div class="mx-2 flex flex-row items-center gap-x-2">
```

Comment:
This fixes a small alignment issue on the conferences page.

### Before
<img width="1038" alt="Screen Shot 2022-05-26 at 7 03 10 PM" src="https://user-images.githubusercontent.com/1526888/170614241-d17bd128-90e6-4545-8549-1bcfbba5808a.png">

### After
<img width="1039" alt="Screen Shot 2022-05-26 at 7 02 48 PM" src="https://user-images.githubusercontent.com/1526888/170614265-fcd6fbe8-38b6-428c-9ca0-6c2cf526f246.png">